tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/Stylesheets.spec.ts

```diff
@@ -0,0 +1,184 @@
+﻿import {ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+
+test.describe('Stylesheets tests', () => {
+
+  const styleSheetName = 'TestStyleSheetFile';
+  const styleSheetFileName = styleSheetName + ".css";
+  const ruleName = 'TestRuleName';
+  const styleFolderName = 'TestStylesheetFolder';
+  
+  test.beforeEach(async ({umbracoUi}) => {
+    await umbracoUi.goToBackOffice();
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+  });
+  
+
+  test('can create a stylesheet file', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.stylesheet.ensureNameNotExists(styleSheetFileName);
+
+    //Act
+    await umbracoUi.stylesheet.openActionsMenuAtRoot();
+    await umbracoUi.stylesheet.clickNewStylesheetFileButton();
+    await umbracoUi.stylesheet.enterStylesheetName(styleSheetName);
+    await umbracoUi.stylesheet.clickSaveButton();
```

Comment:
```suggestion
    await umbracoUi.waitForTimeout(1000);
    await umbracoUi.stylesheet.clickSaveButton();
```

Reply:
waitForTimeout is unnecessary here as the test still works fine without it

Reply:
This test is failing for me when I run it 🙁

Reply:
Ok, we add it for now and when frontend validation is implemented, we will remove it